### PR TITLE
FF fix

### PR DESCRIPTION
**Pull request by LucaMele**

### CODE REVIEW

File: src/app/app.js [hunk 1]
```diff
@@ -64,19 +64,22 @@ document.addEventListener('DOMContentLoaded', () => {
...
-    let lastEnabled = null;
-    const toggleButtons = section.querySelectorAll('.js--toggle');
-    const arr = Array.from(toggleButtons);
-    [lastEnabled] = arr;
-    arr.forEach((button) => {
-      button.addEventListener('click', () => {
-        disable(lastEnabled, section);
-        enable(button, section);
-        lastEnabled = button;
+  // let the components render first, thats why this hack
```
Comment (AndyOGo) [inline src/app/app.js]: what about using event delegation here instead? by `on` module

File: src/components/a-checkbox/index.js [hunk 2]
```diff
@@ -145,4 +145,6 @@ class Checkbox extends BaseComponentGlobal {
   }
 }
 
-window.customElements.define('axa-checkbox', Checkbox);
+document.addEventListener('DOMContentLoaded', () => {
```
Comment (AndyOGo) [inline src/components/a-checkbox/index.js]: I think it's time for a `ready` function

**Comment by LucaMele** [comment]
I cant find any reference for the workaround. Logically make sense that dom should be ready before defining a new custom element. I dont have time to analyse now why ff behaves like that. For me, make sense to wait till dom is loaded so this is good enough

File: src/components/a-checkbox/index.js (hunk 2 above)
Comment (LucaMele) [reply src/components/a-checkbox/index.js]: Will do

File: src/app/app.js (hunk 1 above)
Comment (LucaMele) [reply src/app/app.js]: i think is ok with settimeout as its not critical code. Its just executed before the Components cause their DOM Ready listener is later in the queue. I can comment it better thought so is clear why

**Comment by LucaMele** [comment]
yes we are

**Pull request merged**